WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyRing/PrivacyRingViewModel.cs

```diff
@@ -39,53 +40,60 @@ public PrivacyRingViewModel(WalletViewModel walletViewModel)
 
 		PreviewItems.Add(PrivacyTile);
 
-		_itemsSourceList
+		SetupCancel(enableCancel: true, enableCancelOnEscape: true, enableCancelOnPressed: true);
+	}
+
+	public PrivacyControlTileViewModel PrivacyTile { get; }
+
+	public ObservableCollectionExtended<PrivacyRingItemViewModel> Items { get; } = new();
+	public ObservableCollectionExtended<IPrivacyRingPreviewItem> PreviewItems { get; } = new();
+
+	public Wallet Wallet { get; }
+
+	[System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "Uses DisposeWith()")]
+	protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disposables)
+	{
+		var itemsSourceList = new SourceList<PrivacyRingItemViewModel>();
```

Comment:
SourceList is disposable.
```suggestion
		var itemsSourceList = new SourceList<PrivacyRingItemViewModel>();
		itemsSourceList.DisposeWith(disposables);
```